src/browser-lib/single-spa-angular.ts

```diff
@@ -55,6 +55,7 @@ function bootstrap(opts, props) {
   });
 }
 
+let routingListener;
```

Comment:
Although uncommon, there are some people who use the same instance of the single-spa-angular library for multiple angular applications.

For those people, creating a shared variable for all of them is problematic. Instead, let's add `routingListener` to `opts`, which is the object that is specific to each angular application.